monsteer/plugin/spikeReport.cpp

```diff
@@ -113,6 +115,12 @@ brion::Spikes SpikeReport::read(float min)
 {
     brion::Spikes spikes;
 
+    // First empty the buffered data with no timeout no matter the timestamps
+    // received. This gives the opportunity to update the end time without
+    // having to use a minimum timestamp that may cause the client to block.
+    while( _subscriber->receive( 0 ))
```

Comment:
Good question. It's not absolutely necessary, but maybe it's a good idea for consistency.